about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Exquisite Residence</title>
+  </head>
+  <body>
+    <nav>
+      <ul class="icon">
+        <li>
+          <a href="#"
+            ><img class="social_icon_f" src="./images/facebook1.jpg" alt=""
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img class="social_icon_t" src="./images/twitter.png" alt=""
+          /></a>
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Layout</li>
+      </ul>
+      <div class="nav_main">
+        <a href="./index.html"
+          ><img class="logo" src="./images/logo.png" alt="#"
+        /></a>
+        <ul class="nav_menu">
+          <li class="nav_menu_item close_button">&times;</li>
+          <li class="nav_menu_item">
+            <a class="other_item" href="./index.html">Home</a>
+          </li>
+          <li class="nav_menu_item">
+            <a class="other_item link" href="./about.html">About</a>
+          </li>
+          <li class="nav_menu_item">
+            <a class="other_item" href="#">Program</a>
+          </li>
+          <li class="nav_menu_item"><a class="other_item" href="#">Join</a></li>
+          <li class="nav_menu_item">
+            <a class="other_item" href="#">Sponsor</a>
+          </li>
+          <li class="nav_menu_item"><a class="other_item" href="#">News</a></li>
+          <li class="nav_menu_item">
+            <a class="other_item er_link" href="#">ER Exhibition</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <img src="./images/hamburger-menu.png" alt="hambuger" />
+        </div>
+      </div>
+    </nav>
+    <section class="about">
+      <h3 class="about_sub_title">"Homes For the Masses"</h3>
+      <h1 class="about_title">AFFORDTABLE HOUSING EXHIBITION 2022</h1>
+      <p class="about_intro">
+        Exquisite Residence ehibition always provide a platforn for networking
+        between prospective housing owners, investors, financial experts,
+        marketer construction expert e.t.c.. It create synergy between between
+        all participant. This Year, we have gone a step further beyond the
+        shores of Nigeria a bring experts from different cultural background

```

Comment:
- [ ] Please increase the font size for text content in the about page and make sure your design matches with the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169278810-dc2c60cf-9c3a-4975-aa11-93a8f6fca0e5.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/169278870-1e7a847e-d392-428d-aff5-23d0f2c64670.png)